user-interface/frontend/themes/datamanager/components/main.css

```diff
@@ -81,11 +81,19 @@
 }
 
 .main.legal-notice {
-  padding: var(--lumo-space-m);
+  padding: var(--lumo-space-xl);
+  height: fit-content;
```

Comment:
NIT: This is unnecessary since the height is already defined by the outer main layout. 
```suggestion
```